### PR TITLE
Update team deletion confirmation modal

### PR DESCRIPTION
**Pull request by gtsiolis**
## Description

Update team deletion modal to reflect changes in the product.

## How to test
1. Create a team
2. Try to delete a team
3. Check the confirmation modal is accurate

## Release Notes

```release-note
Update team deletion confirmation modal
```

### CODE REVIEW

File: components/dashboard/src/teams/TeamSettings.tsx [hunk 1]
```diff
@@ -38,46 +38,66 @@ export default function TeamSettings() {
...
-            buttonDisabled={teamSlug !== team!.slug}
-            visible={modal}
-            warningText="Warning: This action cannot be reversed."
-            onClose={close}
-            onConfirm={deleteTeam}
-        >
-            <p className="text-base text-gray-500">You are about to permanently delete <b>{team?.slug}</b> including all associated data with this team.</p>
-            <ol className="text-gray-500 text-m list-outside list-decimal">
-                <li className="ml-5">All <b>projects</b> added in this team will be deleted and cannot be restored afterwards.</li>
-                <li className="ml-5">All <b>workspaces</b> opened for projects within this team will be deleted for all team members and cannot be restored afterwards.</li>
```
Comment (gtsiolis) [inline components/dashboard/src/teams/TeamSettings.tsx:76]: **fyi**: This is the only code change (removal), the rest were automatically formatting changes.

Comment (gtsiolis) [reply components/dashboard/src/teams/TeamSettings.tsx:76]: Cc @jankeromnes @geropl based on the comment in https://github.com/gitpod-io/gitpod/issues/8760#issuecomment-1065546765.

**Review by easyCZ — DISMISSED** [review]
This looks good to me, but will need a rebase.

**Comment by gtsiolis** [comment]
Thanks for taking a look, @easyCZ!

Rebased, waiting for build to finish, and will merge afterwards. 🏓

**Comment by gtsiolis** [comment]
@easyCZ could you approve again to unblock merging this?

**Review by easyCZ — APPROVED** [review]

**Comment by gtsiolis** [comment]
Díky, @easyCZ! 🍻

**Pull request merged**